src/Debugging-Utils/Halt.class.st

```diff
@@ -184,24 +193,36 @@ Halt class >> onCountWithBehavior: behaviorBlock [
 Halt class >> once [
 	"Stop once and only once the execution of the method containing the expression Halt once. If you need to get it stops another time after the first stop, use Halt resetOnce."
 
-	| callingContext senderContext node |
-	<debuggerCompleteToSender>
```

Comment:
Maybe the pragma should be kept, because this is used to hint the debugger that this method should be filtered from the shown stack by default (and usually we want to stop at the caller, not here right?

Or maybe it's not needed? What does the debugger show when this halts without the pragma?

Reply:
I mean, if you do `Halt once` directly in a method

Reply:
Tx I thought that I kept it so I will added back